src/gui/torrentcontentmodel.cpp

```diff
@@ -91,14 +91,18 @@ namespace
         QIcon icon(const QFileInfo &info) const override
         {
             SHFILEINFO sfi = { 0 };
-            HRESULT hr = ::SHGetFileInfoW(info.absoluteFilePath().toStdWString().c_str(),
+            QString nativePath = Utils::Fs::toNativePath(info.absoluteFilePath());
```

Comment:
I don't know if the extension only will work. But I think filename+ext should work. I'll test tomorrow.